build/Dockerfile.base

```diff
@@ -1,18 +0,0 @@
-FROM ubuntu:16.04
```

Comment:
Also, this image is used for running tests and go checks. https://github.com/contiv/auth_proxy/blob/28b32e91762d363ac41569efc8fe7988a4488ece/scripts/checks.sh#L7 
https://github.com/contiv/auth_proxy/blob/28b32e91762d363ac41569efc8fe7988a4488ece/scripts/systemtests.sh#L34
https://github.com/contiv/auth_proxy/blob/28b32e91762d363ac41569efc8fe7988a4488ece/scripts/unittests.sh#L26

If we're removing this, it's needs to propagated everywhere.